scripts/gen_vapid_keys.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* scripts/gen_valid_keys.js creates public and private keys suitable for
+   use with VAPID in push notifications.
+
+   Usage:
+   ./scripts/gen_valid_keys.js [filename]
+
+   The filename will default to:
+
+       ./config/vapid-keys.json
+
+   If this file already exists, this script will show an error message
+   and exit. You must remove the file if you want to generate a new
+   keypair.
+*/
+
+const fs = require('fs')
+const assert = require('assert')
+const config = require('../config')
+const webpush = require('web-push')
+
+const vapidKeysFile = config.get('vapidKeysFile')
+
+try {
+  var fileExists = fs.existsSync(vapidKeysFile)
+  assert(!fileExists, 'keys file already exists')
+} catch(e) {
+  process.exit()
```

Comment:
This logic was copied from gen_keys.js, and IIUC it's intentionally not a failure - it just prints "keys exist" and exits, so you can safely run the script multiple times without clobbering your keys.  I'm going to change it to use a plain `if` rather than this `assert` business.